jstest/testrunner/devices/artik053.py

```diff
@@ -23,6 +23,9 @@ class ARTIK053Device(RemoteDevice):
     '''
     Device of the ARTIK053 target.
     '''
+    # Override the default properties from RemoteDevice so that pylint doesn't error.
+    workdir = channel = None
```

Comment:
You should move these variables into the `__init__` function as `self.workdir` and `self.channel`.